php/php.dbgp/src/org/netbeans/modules/php/dbgp/breakpoints/BreakpointModel.java

```diff
@@ -110,8 +111,14 @@ public String getIconBase(Object node) throws UnknownTypeException {
             AbstractBreakpoint breakpoint = (AbstractBreakpoint) node;
             if (!breakpoint.isEnabled()) {
                 return DISABLED_BREAKPOINT;
+            } else {
+                VALIDITY validity = breakpoint.getValidity();
+                if (validity.equals(VALIDITY.VALID) || validity.equals(VALIDITY.UNKNOWN)) {
```

Comment:
Cannot we improve this check? I mean, perhaps the Validity enum itself could have isValid() method?

I am just thinking what will happen if we add a new Validity constant, whether we will handle all the usages.

Reply:
`VALIDITY` is API.
https://github.com/apache/netbeans/blob/2141d37d045da9d247eb266d4c5f008255312128/ide/api.debugger/src/org/netbeans/api/debugger/Breakpoint.java#L53

Perhaps, we should have the `isValid()` method in the [`Utils`](https://github.com/apache/netbeans/blob/master/php/php.dbgp/src/org/netbeans/modules/php/dbgp/breakpoints/Utils.java) class instead?


I found the 2 same conditions.

https://github.com/apache/netbeans/blob/2141d37d045da9d247eb266d4c5f008255312128/php/php.dbgp/src/org/netbeans/modules/php/dbgp/annotations/BrkpntAnnotation.java#L45

https://github.com/apache/netbeans/blob/2141d37d045da9d247eb266d4c5f008255312128/php/php.dbgp/src/org/netbeans/modules/php/dbgp/breakpoints/BreakpointModel.java#L103

Reply:
I see, thanks. Yes, in such a case, I would prefer a method, especially if we already have it in more places.

Reply:
Added the `isValid()` method to the `Utils` class.

Reply:
Thanks!